Assignment_2-data-emma-luk.js

```diff
@@ -0,0 +1,101 @@
+﻿// Emma Luk
+// Assignments 2 Using the revealing module pattern, build a Date utility module 
+
+var myDate = (function iife() {
+    var displaysModuleDate = null;
+
+    //private function: date instantiation
+    function _initDate() {
+        return new Date();
+    }
+
+    //check here that date is set: if not, initialize date to now
+    function setDate(date) {
+        if (date === undefined && null) {

```

Comment:
This condition will always evaluate to false. If you want to check a value against two different values, you have to check the value explicitly each time. Additionally, you'd want to use the || operator as a value cannot be both undefined and null since null is a defined value.  `if (date === undefined || date === null)`